src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -12,10 +12,10 @@
 <HTML>Could_not_find_file_'%0'<BR>linked_from_entry_'%1'</HTML>=<HTML>Le_fichier_'%0'_n'a_pas_pu_être_trouvé_<BR>à_partir_du_lien_de_l'entrée_'%1'</HTML>
 <select>=<sélectionner>
 <select_word>=<entrer_le_mot-clef>
-Abbreviate_journal_names_of_the_selected_entries_(ISO_abbreviation)=Abréger_les_noms_de_journaux_des_entrées_sélectionnées_(abréviations_ISO)
-Abbreviate_journal_names_of_the_selected_entries_(MEDLINE_abbreviation)=Abréger_les_noms_de_journaux_des_entrées_sélectionnées_(abréviations_MEDLINE)
+Abbreviate_journal_names_of_the_selected_entries_(ISO_abbreviation)=Abréger_les_noms_de_journAUX_des_entrées_sélectionnées_(abréviations_ISO)
```

Comment:
the replacement here and at a lot of other places in this file is wrong.

Please recheck the changes in this file!